src/Components/Team/teamData.js

```diff
@@ -0,0 +1,84 @@
+export const teamData = [
```

Comment:
We can have a practice to make it a json file.
It makes it easier for us to write test cases in future.
It's a good practice.